tox.ini

```diff
@@ -30,12 +30,8 @@ whitelist_externals = rm
 [testenv:lint]
 deps =
      {[base]testdeps}
-     pycodestyle
-     pydocstyle
      pylint
 commands =
-    pycodestyle {[base]name} --exclude tests
-    pydocstyle {[base]name}
     pylint {[base]name} --ignore tests
```

Comment:
pylint must be able to import the package and its dependencies so it is not suitable for pre-commit.
It is possible to make it work locally but not in CI (at least not easily): https://pylint.pycqa.org/en/latest/user_guide/pre-commit-integration.html

Reply:
Not sure I follow, what isn't working?

Reply:
`Pylint` does not work with pre-commit (because it is not static).